### PR TITLE
tvOS support for CocoaPods + Carthage support

### PR DESCRIPTION
**Pull request by marcvanolmen**

### CODE REVIEW

**Comment by f2prateek** [comment]
Thanks! I'm not too familiar with Carthage and tvOS yet, could you comment with instructions so I can test this out manually on our end as well to verify everything is working as expected?

**Comment by marcvanolmen** [comment]
## Cocoapods:

For cocoapod the support was easy because just adding one extra line for the new platform tvOS, in the spec file.

## Carthage.

I assume you have Homebrew installed on your machine:

```
brew install carthage
```

Then git clone the proejct.

```
cd analytics-swift
carthage build --no-skip-current
```

This will create a folder Carthage and inside it will places the result of the framework for tvOS, iOS and OS X. This proves that your project can be successfully build with carthage.  

Basically what we did to support Carthage is to add 2 extra targets to the Xcode.project that builds a framework for iOS and tvOS. Carthage just uses  Xcode projects targets to build frameworks. Those targets needs to build with schemes that are shared in your Xcode project.

**Comment by f2prateek** [comment]
Sorry for the delay in responding, I did test this out a originally but I couldn't integrate into a real app after building the library. I don't have the error I encountered with me right now, but I'll try to repeat it and share it once I get the chance to.

**Comment by f2prateek** [comment]
Just tested this again and seems to work, thanks again!
